plugin/pkg/scheduler/algorithm/priorities/priorities.go

```diff
@@ -82,6 +99,35 @@ func calculateResourceOccupancy(pod *api.Pod, podRequests *schedulercache.Resour
 	}
 }
 
+// Calculate the resource used on a node.  'node' has information about the resources on the node.
+// 'pods' is a list of pods currently scheduled on the node.
+// TODO: Use Node() from nodeInfo instead of passing it.
+func calculateUsedPriority(pod *api.Pod, podRequests *schedulercache.Resource, node *api.Node, nodeInfo *schedulercache.NodeInfo) schedulerapi.HostPriority {
```

Comment:
Can you give a simple example where the approach you're using here would give a different final sort order for the nodes, compared to just reversing the order given by calculateUnusedPriority()?